.python-version

```diff
@@ -0,0 +1 @@
+3.11
```

Comment:
Good idea to explicitly define our Python version for contributors that use `pyenv` to manage Python environments.

Reply:
good idea